src/openapi/transport/auth.js

```diff
@@ -185,12 +192,19 @@ TransportAuth.prototype.incrementErrorCounter = function(url) {
  * @param {string} url - The url/endpoint for which error count is returned.
  * @returns {number} The number of errors
  */
-TransportAuth.prototype.getUrlErrorCount = function(url) {
-    if (this.authorizationErrorCount.hasOwnProperty(url)) {
-        return this.authorizationErrorCount[url];
+TransportAuth.prototype.areUrlAuthErrorsProblematic = function(
+    url,
+    authExpiry,
+) {
+    if (this.authorizationErrors.hasOwnProperty(url)) {
+        for (let i = 0; i < this.authorizationErrors[url].length; i++) {
+            if (this.authorizationErrors[url][i].authExpiry !== authExpiry) {
+                return true;
+            }

```

Comment:
so we will log error if request with same url with different token failed ?
may be we should update error message as well ?

Reply:
yes. updated message. its to stop one problematic service making us ddos the token request url.